NEWS.md

```diff
@@ -1,5 +1,7 @@
 # purrr (development version)
 
+* Added a new keep_empty argument to list_c(), list_cbind() and list_rbind() which will keep empty elements as NA in the returned dataframe. (#1096)
```

Comment:
```suggestion
* Added a new keep_empty argument to `list_c()`, `list_cbind()`, and `list_rbind()` which will keep empty elements as `NA` in the returned data frame. (#1096)
```

Reply:
If you like, you can add your GitHub handle after the issue: `(#1096, @SokolovAnatoliy)` .